### DISCUSSION TITLE
VSCode Cadence Extension Still Broken on Windows

### DISCUSSION THREAD

**Discussion by jacob-tucker** [onflow/flow#1228] [Developer Experience] [answered]
Hey hey, I saw on Twitter that the VSCode Extension was fixed for Windows. However it is still broken for me. Contracts do not get imported correctly because the extension is having trouble with local file paths.

**Comment by DylanTinianov** [comment]
Hey Jacob, the fixes have not been released yet. Not sure who mentioned on Twitter that it was fixed. Still in the testing phase, but we plan on doing a release early next week!

You can check out the progress here: [#201](https://github.com/onflow/vscode-cadence/issues/201)

**Reply by fotescodev** [reply]
@jacob-tucker my bad, I misunderstood. Thanks for clarifying @DylanTinianov.

**Comment by DylanTinianov** [comment, marked as answer]
This has been solved 🥳

**Reply by jacob-tucker** [reply]
Yessssssssss!!!
